core/embed/trezorhal/unix/common.c

```diff
@@ -28,6 +28,8 @@
 #include "display.h"
 #include "memzero.h"
 
+void __attribute__((noreturn)) main_clean_exit();
+
```

Comment:
We can get rid of this now.

Reply:
```
embed/trezorhal/unix/common.c: In function ‘emulator_event_filter’:
embed/trezorhal/unix/common.c:52:7: error: implicit declaration of function ‘main_clean_exit’ [-Werror=implicit-function-declaration]
   52 |       main_clean_exit();
```

Reply:
Oh, sorry, I overlooked this. So everything is ok.